emission/analysis/modelling/trip_model/config.py

```diff
@@ -75,5 +75,9 @@ def get_minimum_trips():
         raise TypeError(msg)
     return minimum_trips
 
-
-
+def get_maximum_stored_model_count():
```

Comment:
I'm not going to hold up the merge for this release, but would be good to avoid these extraneous whitespace changes in future PRs

Reply:
Oh, right. It's just that sometimes there are extra lines at the end of the file and when I'm adding new code I tend to keep the spacing between modules similar to how it is in other parts of the file. But will try to minimize these changes.